opsramp/rba.py

```diff
@@ -46,6 +46,17 @@ def create(self, name, parent_uuid=None):
     def category(self, uuid):
         return Category(self, uuid)
 
+    # Updates an RBA category based on its ID.
+    def update(self, uuid, definition):
+        # Making sure the UUID is specified in the request body as it's not
+        # specified in the URL for some reason.
```

Comment:
that's classic opsramp, there are inconsistencies all over the place. the typical pattern for update, as you said, would be POST to url/uuid but just for variety this one has PUT and embeds the id in the payload. Lovely.